editor/src/components/canvas/canvas-strategies/strategies/reorder-slider-strategy.spec.browser2.tsx

```diff
@@ -613,6 +617,26 @@ describe('Reorder Slider Strategy Control', () => {
 
     await selectAndHover('utopia-storyboard-uid/scene-aaa/app-entity:container/ccc', renderResult)
 
+    const targetControl = renderResult.renderedDOM.queryByTestId('reorder-slider-control')
+    expect(targetControl).toBeNull()
+  })
+  it('the reorder control is not visible on simple, 1d flow layouts for metadata with out of order children', async () => {
```

Comment:
> for metadata with out of order children'

I don't understand, are these children out of order? not in the code and DOM right? only in the metadata, right?

Reply:
Sorry this is a good point, I'll clarify the naming of the test. It's that the order returned by calling `Object.keys` on the metadata doesn't match the code or DOM order.